### PR TITLE
Edit quiz

### PR DESCRIPTION
**Pull request by pachia-lee**
Edit quiz feature works now

### CODE REVIEW

File: Models/BusinessLogic.cs [hunk 1]
```diff
@@ -80,29 +81,63 @@ public async Task SkipLogin() {
...
 
     #region Quizzes
+    public QuizManager? EditQuizManager { get; set; }
+
+    public QuizManager? getEditQuizManager() {
+        return (EditQuizManager != null) ? EditQuizManager : null;
+    }
+    public void SetEditQuestionQuizManager(QuizManager quizManager) {
+        EditQuizManager = quizManager;
+    }
```
Comment (benderje26) [inline Models/BusinessLogic.cs:128]: Could getEditQuizManager() and SetEditQuizManager be done with just the property

File: Models/IBusinessLogic.cs [hunk 2]
```diff
@@ -32,6 +32,9 @@ public interface IBusinessLogic : INotifyPropertyChanged {
     /// <returns>The result and a nullable string showing the message if something went wrong</returns>
     Task<(LogoutResult, string?)> Logout();
 
+    public QuizManager? getEditQuizManager();
+    void SetEditQuestionQuizManager(QuizManager quizManager);
```
Comment (benderje26) [inline Models/IBusinessLogic.cs]: Like earlier mentioned, we could put that property also in the IBusinessLogic which might simplify it and then just get rid of the methods

**Review by benderje26 — DISMISSED** [review]
Nice job! I appreciate you making this. If you could just consider what was mentioned that would be great

File: Models/IBusinessLogic.cs (hunk 2 above)
Comment (pachia-lee) [reply Models/IBusinessLogic.cs]: Thank you for your input! I've deleted the setter and getter method.

**Review by skogmanpe40 — APPROVED** [review]
Looks good

**Pull request merged**